ksql-execution/src/main/java/io/confluent/ksql/execution/transform/KsqlProcessingContext.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.execution.transform;
+
+/**
+ * Context passed to transformers
```

Comment:
I think this could use more detailed java docs. What exactly is this context used for? Is it updated per row? What should be  in here and what should not?

Reply:
done. Though it's still forming at the moment.